OpenRA.Game/Network/GeoIP.cs

```diff
@@ -11,63 +11,124 @@
 
 using System;
 using System.IO;
+using System.Linq;
 using System.Net;
-using ICSharpCode.SharpZipLib.GZip;
-using MaxMind.Db;
+using System.Net.Sockets;
+using System.Numerics;
+using ICSharpCode.SharpZipLib.Zip;
 
 namespace OpenRA.Network
 {
 	public class GeoIP
 	{
-		public class GeoIP2Record
+		class IP2LocationReader : IDisposable
 		{
-			[Constructor]
-			public GeoIP2Record(GeoIP2Country country)
+			public readonly DateTime Date;
```

Comment:
Not used.

Reply:
We'll need this when implementing the next part (having the server download and manage the database), so i'd prefer if we can keep this here for now.